print_designer/hooks.py

```diff
@@ -12,7 +12,7 @@
 
 # include js, css files in header of desk.html
 app_include_js = "pdf.worker.bundle.js"
-app_include_css = "pdfjs.bundle.css"
+app_include_css = ["pdfjs.bundle.css", "print.bundle.css"]
```

Comment:
Why is this loaded on ALL pages of desk? Please load these things async else it will cause weird behaviour sooner or later.

Reply:
yeah, this is just to improve UX for now will async. also this is required for print view and loading it on load is kind of slow

Reply:
only very first load will be slow, after that it's cached (can be done even without localstorage)